releases/release-6.1.1.md

```diff
@@ -0,0 +1,121 @@
+    (dup: release-6.2.0.md > Bug fixes> TiKV)- Fix the issue that the Commit Log Duration of a new Region is too high, which causes QPS to drop [#13077](https://github.com/tikv/tikv/issues/13077)
+    (dup: release-6.2.0.md > Improvements> TiKV)- Support dynamically modifying the number of sub-compaction operations performed concurrently in RocksDB (`rocksdb.max-sub-compactions`) [#13145](https://github.com/tikv/tikv/issues/13145)
+
++ PD
+
+    - Fix the issue that the online process is not accurate when having invalid label settings. [#5234](https://github.com/tikv/pd/issues/5234)
+    - grpc: fix the wrong error handler [#5373](https://github.com/tikv/pd/issues/5373)
+    - Fix the issue that `/regions/replicated` may return the wrong status [#5095](https://github.com/tikv/pd/issues/5095)
+
++ TiFlash
+
+    (dup: release-5.4.2.md > Bug Fixes> TiFlash)- Fix the issue that TiFlash crashes after dropping a column of a table with clustered indexes in some situations [#5154](https://github.com/pingcap/tiflash/issues/5154)
+    - Fix the issue that format throw data truncated error [#4891](https://github.com/pingcap/tiflash/issues/4891)
+    - fix the problem that there may be some obsolete data left in storage which cannot be deleted [#5659](https://github.com/pingcap/tiflash/issues/5659)
+    - Reduce unnecessary CPU usage in some edge cases [#5409](https://github.com/pingcap/tiflash/issues/5409)
+    - Fix a bug that TiFlash can not work in a cluster using ipv6 [#5247](https://github.com/pingcap/tiflash/issues/5247)
+    - Fix a panic issue in parallel aggregation when an exception is thrown. [#5356](https://github.com/pingcap/tiflash/issues/5356)
+
++ Tools
+
+    + TiCDC
+
+        - Fix the wrong maximum compatible version number [#6039](https://github.com/pingcap/tiflow/issues/6039)
+        - Fix a bug that may cause cdc server panic if it received a http request before cdc server fully started. [#5639](https://github.com/pingcap/tiflow/issues/5639)
+        - Fix ddl sink panic when changefeed syncpoint is enable. [#4934](https://github.com/pingcap/tiflow/issues/4934)
+        - Fix a data race in black hole sink. [#5714](https://github.com/pingcap/tiflow/issues/5714)
+        - Fix a bug that causes get changefeeds api does not well after cdc server restart. [#5837](https://github.com/pingcap/tiflow/issues/5837)
+        - Fix a data race in black hole sink. [#6206](https://github.com/pingcap/tiflow/issues/6206)
+        - Fix TiCDC panic issue when disable the old value of changefeed [#6198](https://github.com/pingcap/tiflow/issues/6198)
+
+    + Backup & Restore (BR)
+
+        (dup: release-5.4.2.md > Bug Fixes> Tools> Backup & Restore (BR))- Fix a bug that BR reports `ErrRestoreTableIDMismatch` in RawKV mode [#35279](https://github.com/pingcap/tidb/issues/35279)
+
+    + Dumpling
+
+        - use net.JoinHostPort to generate host-port part of URI [#36112](https://github.com/pingcap/tidb/issues/36112)
+
+    + TiDB Lightning
+
```

Comment:
- fix connect to tidb when using ipv6 host [#35880](https://github.com/pingcap/tidb/issues/35880)
- add ReadIndexNotReady as retryable ingest error [#36566](https://github.com/pingcap/tidb/issues/36566)
- hide sensitive log for server mode lightning [#36374](https://github.com/pingcap/tidb/issues/36374)